### PR TITLE
View vars will now tell you if something has been previously var edited

### PR DESCRIPTION
**Pull request by Incoming5643**
A quick and dirty way to show that a [literally anything] in the game has been admin touched, and thus ineligible to be referenced in bug reports.

Might also have some uses in persistence based mechanics where actual objects are saved instead of just paths.

### CODE REVIEW

File: code/modules/admin/verbs/modifyvariables.dm [hunk 1]
```diff
@@ -5,7 +5,7 @@ var/list/forbidden_varedit_object_types = list(
 										/datum/admin_rank					//editing my own rank? it's more likely than you think
 									)
 
-var/list/VVlocked = list("vars", "client", "virus", "viruses", "cuffed", "last_eaten", "unlock_content", "step_x", "step_y", "force_ending")
+var/list/VVlocked = list("vars", "var_edited", "client", "virus", "viruses", "cuffed", "last_eaten", "unlock_content", "step_x", "step_y", "force_ending")
```
Comment (Incoming5643) [inline code/modules/admin/verbs/modifyvariables.dm]: This is technically redundant, as any attempts to set var_edited to 0 again would set it to 1 as it is, in itself, a var edit.

It's more just to keep the var list a little tidier.

**Comment by Incoming5643** [comment]
Conflicts, how'd someone manage that?

File: code/modules/admin/verbs/modifyvariables.dm (hunk 1 above)
Comment (Shadowlight213) [inline code/modules/admin/verbs/modifyvariables.dm]: VVlocked just locks stuff to needing +DEBUG to edit. it doesn't hide it from the list.

Comment (Incoming5643) [inline code/modules/admin/verbs/modifyvariables.dm]: oh well, in that case it's just here to mock people who try to edit it

Comment (Shadowlight213) [inline code/modules/admin/verbs/modifyvariables.dm]: Except that everyone but admin candidates and below has +DEBUG currently :P
Wont have any effect.

**Pull request merged**